sros2/package.xml

```diff
@@ -22,6 +22,7 @@
   <test_depend>ament_mypy</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
+  <test_depend>ros_testing</test_depend>
```

Comment:
I don't see the "ros_testing" string anywhere in this PR. Is this dependency being used ?

Reply:
Same as above.